src/app/routes/welcome/welcome.component.ts

```diff
@@ -47,6 +47,7 @@ export class WelcomeComponent implements OnInit {
   }
 
   async connectToEwKeyManager() {
+    return;
```

Comment:
very strange changes, don't we need "await this.connectToWallet(WalletProvider.EwKeyManager);" ?